models/issues/comment.go

```diff
@@ -197,6 +197,19 @@ func (t CommentType) HasMailReplySupport() bool {
 	return false
 }
 
+func (t CommentType) CountedAsConversation() bool {
+	for _, ct := range ConversationCountedCommentType() {
+		if t == ct {
+			return true
+		}
+	}
+	return false
+}
+
+func ConversationCountedCommentType() []any {
```

Comment:
```any``` ?

Reply:
It can be used directly in format functions but `[]CommentType` cannot.

Reply:
17d8a24801dfd5af8cec6efcdb3c78050155203f

Reply:
Why not use `statsQuery( ... builder....In("type", ConversationCountedCommentType) ... )`?